content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

```diff
@@ -0,0 +1,64 @@
+You will receive the notification if the device in question is enabled in the **Notification Preference** option. To check the preference option, refer to [Notification-Preference](/support/how-to/notification prefernce).
+ {{</accordion>}}
+
+{{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}
+Create a new ticket in the fabric ticketing app under the category **Account** and select **User Management** as the account type. Contact your Admin or follow up with the support team or Account Manager if you are unable to log into the portal.
+{{</accordion>}}
+
+### Permission
+
+{{<accordion title="I am unable to upload documents and tried multiple times on multiple tickets." col="in" href="accordion1">}}
+Opt in to the ticketing beta using the ticketing drop down in the global navigation.
+{{</accordion>}}
+
+{{<accordion title="I have raised ticket using Rackspace BETA version of ticketing system, but unable to edit the ticket. I am getting the notification as I don't have permission to update." col="in" href="accordion2">}}
+Customer should raise the ticket in one of those 5 categories (Change/Incident/Account/Billing and payments/Request for information) and select appropriate account and product. 
+Also need to contact admin (customer side) or need to contact their Customer Success Manager or Support Team.
+{{</accordion>}}
+
+{{<accordion title="I need to access the Managed Backup option in the portal." col="in" href="accordion3">}}
+To access **View Reports**, you can see documentation linked below. To have access to the Managed Backup feature. https://one.rackspace.com/display/myrackspace/Managed+Backup?searchId=OLV66F2LT
+{{</accordion>}}
+
+{{<accordion title="I am having issue in loading MAR report in the portal." col="in" href="accordion4">}}
+MAR is generated at Global Data Operations side monthly which is then reported to MyRack. 
+Below is a documentation with more details:
+[MAR Process](https://one.rackspace.com/display/SMBHAMS/MAR+Processes)
+{{</accordion>}}
+
+{{<accordion title="How can I update device name in Vcenter by removing Devices IDs?" col="in" href="accordion5">}}
+The naming convention for vm should be <CoreDeviceId>-<device name>, for these vms seems this pattern is not followed so we observed that deviceid is null in our database.
+{{</accordion>}}
+
+{{<accordion title="I am not able to generate PIN via Rackspace portal." col="in" href="accordion6">}}
+
+The customer should be able to unlock the PIN from the portal, according to the directions posted here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/) if they cannot, I would advise contacting the customer identity team, as that is where the PIN is stored. Be advised though: This is a security measure.  No action should be taken to subvert it without a suitable verification of the customers identity.
```

Comment:
```suggestion
The customer should be able to unlock the PIN from the portal, according to the directions posted here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/) if they cannot, then it is advised to contact the customer identity team, as that is where the PIN is stored. Be advised though: This is a security measure.  No action should be taken to subvert it without a suitable verification of the customers identity.
```

Reply:
Is this FAQ for Customers? Seems like the answer is speaking to a Racker.

Reply:
Updated as "According to the instructions here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/), you should be able to unlock the PIN via the portal. If you are unable to do so, you should contact the customer identity team, which is where the PIN is kept. However, bare in mind that this is a security measure. Without a proper verification of the customer's identity, no action should be performed to subvert it."